docs/blockchain_schema/version1.md

```diff
@@ -0,0 +1,58 @@
+# Version 1 schema
+## Block schema
+```
+{
+   "sequence": [int][block_sequence_no], // 0-indexed
+   "origin_created": [datetime][created_datetime_of_same_block_with_earliest_version], // new
+   "version": 1,
+   "prev_version_block": [hashPtr][ptr_to_same_block_in_previous_version_of_blockchain], // new
+   "prev_hash": [hash][hash_of_the_previous_block], // null for first block
+   "operations": [list][operations_contained_in_block]
+}
+```
+## Operation schema
+```
+{
+    "sequence": [int][operation_seqeuence_no], // 0-indexed
+    "op_code": [string][choice(operation)],
+    "creator_mid": [string][raha_assigned_uid], // new
+    "data": [obj][operation_specific_data_obj]
+}
+```
+
+## Choices
+operation: `["REQUEST_INVITE", "TRUST"]`
+
+## Data objects
+### REQUEST_INVITE `// new`
+```
+{
+    "full_name": [string][full_name],
+    "to_mid": [string][mid_of_user_requesting_invite_from],
+    "video_url": [url][url_of_invite_video],
+}
+```
+
+### Trust `// new`
+```
+{
+    "to_mid": [string][mid_of_user_to_trust],
+}
+```
+
+## Notes
+* A user, Alice, can only TRUST another user if there exists a path of TRUST to Alice from a user, Bob, who become a member of the network in a prior block. Block 0 is exempt from this restriction.
```

Comment:
Should we say `prior operation` instead of `prior block`?

Reply:
I don't think the blockchain currently has any knowledge of how operations should be ordered within a block - the ordering is essentially random from its perspective - so it feels weird to me to make operation acceptance dependent on operation ordering.

That being said.. doesn't the requirement of the existence of a path of trust mean that someone will be accepted as long as they aren't an isolated cluster?